rusoto/core/src/signature.rs

```diff
@@ -55,46 +66,57 @@ impl SignedRequest {
             canonical_uri: String::new(),
         }
     }
+    /// Adds a header of format <"content-type", content-type>
```

Comment:
I think we'll want a new line between this comment, and the closing brace on the line above.

Also I think:
```rust
/// Sets the value of the "content-type" header.
```

Makes more sense